mod/quiz/accessrule/seb/classes/access_manager.php

```diff
@@ -297,4 +385,44 @@ public function should_validate_browser_exam_key() : bool {
             settings_provider::USE_SEB_CLIENT_CONFIG,
         ]);
     }
+
+    /**
+     * Check that at least one browser exam key exists in the quiz settings.
+     *
+     * @return bool True if one or more keys are set in quiz settings.
+     */
+    public function valid_browser_exam_key_exists(): bool {
```

Comment:
Should this method be public? Is it used anywhere outside of that class?
Also the name is a bit confusing for me as I would think it check key in a browser and not setting. Maybe rename to something like is_allowed_browser_examkeys_configured to stress that we are checking it in settings

Reply:
Good point. Changed name, made private, and removed related unit tests.